content/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-strong-password.md

```diff
@@ -17,7 +17,7 @@ You must choose or generate a password for your {% data variables.product.produc
 - 15 characters long with any combination of characters
 
 To keep your account secure, we recommend you follow these best practices:
-- Use a password manager, such as [LastPass](https://lastpass.com/) or [1Password](https://1password.com/), to generate a password of at least 15 characters.
+- Use a password manager, such as [LastPass](https://lastpass.com/), [1Password](https://1password.com/), or [Keeper](https://keepersecurity.com/), to generate a password of at least 15 characters.
```

Comment:
I believe there shouldn't be a comma after the 1Password link.

Reply:
You are correct that it is not necessary grammatically, but we use the Oxford comma (https://brand.github.com/content/grammar#commas)

Reply:
Thanks for the reply @skedwards88, I am now aware! :)

Reply:
add storing-two-factor-codes [Keeper](https://docs.keeper.io/enterprise-guide/storing-two-factor-codes)